codec/src/test/java/io/netty/handler/codec/compression/JZlibTest.java

```diff
@@ -26,4 +28,10 @@ protected ZlibEncoder createEncoder(ZlibWrapper wrapper) {
     protected ZlibDecoder createDecoder(ZlibWrapper wrapper) {
         return new JZlibDecoder(wrapper);
     }
+
+    @Test(expected = DecompressionException.class)
+    @Override
+    public void testZLIB_OR_NONE3() throws Exception {
```

Comment:
Okay. I will leave the feature changes as they are currently. I don't see why gzip should be handled when the wrapper type is ZLIB_OR_NONE. I will also remove the test case - testZLIB_OR_NONE3() as it tests a feature which is not implemented.